pallets/capacity/src/lib.rs

```diff
@@ -1106,24 +1114,20 @@ impl<T: Config> StakingRewardsProvider<T> for Pallet<T> {
 	type AccountId = T::AccountId;
 	type RewardEra = T::RewardEra;
 	type Hash = T::Hash;
+	type Balance = BalanceOf<T>;
```

Comment:
Not only were we not using the defined types but we could also use a new one for Balance.